### DISCUSSION TITLE
Troubleshooting Touch Screen Inaccuracy | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1654] [Q&A]
# Troubleshooting Touch Screen Inaccuracy | Seeed Studio Wiki

reTerminal-FAQ

https://wiki.seeedstudio.com/troubleshooting-touch-screen-inaccuracy/

**Comment by batesenergy** [comment]
This does not seem to work with Wayland / Bookworm. My touchpad is all over the place.

**Comment by maskoba** [comment]
Wayland(labwc):

sudo nano /etc/udev/rules.d/99-touch-calibration.rules

SUBSYSTEM=="input", ATTRS{name}=="Goodix Capacitive TouchScreen", ENV{LIBINPUT_CALIBRATION_MATRIX}="0 1 0 -1 0 1"
